controllers/memberoperatorconfig/configuration.go

```diff
@@ -28,6 +145,52 @@ func (a MemberStatusConfig) RefreshPeriod() time.Duration {
 	return d
 }
 
+type ToolchainClusterConfig struct {
+	t toolchainv1alpha1.ToolchainClusterConfig
+}
+
+func (a ToolchainClusterConfig) HealthCheckPeriod() time.Duration {
+	defaultClusterHealthCheckPeriod := "10s"
+	healthCheckPeriod := getString(a.t.HealthCheckPeriod, defaultClusterHealthCheckPeriod)
+	d, err := time.ParseDuration(healthCheckPeriod)
+	if err != nil {
+		d, _ = time.ParseDuration(defaultClusterHealthCheckPeriod)
+	}
+	return d
+}
+
+func (a ToolchainClusterConfig) HealthCheckTimeout() time.Duration {
+	defaultClusterHealthCheckTimeout := "3s"
+	healthCheckTimeout := getString(a.t.HealthCheckTimeout, defaultClusterHealthCheckTimeout)
+	d, err := time.ParseDuration(healthCheckTimeout)
+	if err != nil {
+		d, _ = time.ParseDuration(defaultClusterHealthCheckTimeout)
+	}
+	return d
+}
+
+type WebhookConfig struct {
+	w toolchainv1alpha1.WebhookConfig
+}
+
+func (a WebhookConfig) Deploy() bool {
+	return getBool(a.w.Deploy, true)
+}
+
+func getBool(value *bool, defaultValue bool) bool {
+	if value != nil {
+		return *value
+	}
+	return defaultValue
+}
+
+func getInt(value *int, defaultValue int) int {
+	if value != nil {
+		return *value
+	}
+	return defaultValue
+}
```

Comment:
Didn't you forget to update it with the functions from common? https://github.com/codeready-toolchain/toolchain-common/pull/196

Reply:
Yes, thanks for reminding me.

Reply:
Done in 1a217f60b6ea7f2ba3ed0a389b0b544eadc26c39